### PR TITLE
Added Social Media (Facebook, Twitter, Instagram) Details

### PR DESCRIPTION
**Pull request by kenpaxtonlim**

### CODE REVIEW

**Review by Zzmobie — COMMENTED** [review]
LGTM

**Review by Zzmobie — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/SocialMedia.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        String toString = "";
+
+        if (!facebook.equals("")) {
+            toString += "FB: " + facebook + " ";
+        }
+        if (!twitter.equals("")) {
+            toString += "TW: " + twitter + " ";
+        }
+        if (!instagram.equals("")) {
+            toString += "IG: " + instagram;
```
Comment (500poundbear) [inline src/main/java/seedu/address/model/person/SocialMedia.java:78]: Missing + " ";

File: src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java [hunk 2]
```diff
@@ -96,6 +97,19 @@ public EditPersonDescriptorBuilder withTags(String... tags) {
...
 
+    /**
+     * Sets the {@code SocialMedia} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withSocialMedia(String facebook, String twitter, String instagram) {
+        try {
+            descriptor.setSocialMedia(ParserUtil.parseSocialMedia(
+                    Optional.of(facebook), Optional.of(twitter), Optional.of(instagram)));
+        } catch (IllegalValueException ive) {
+            throw new IllegalArgumentException("social media usernames is expected to be unique.");
```
Comment (500poundbear) [inline src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java:108]: is it are or is here?

**Review by 500poundbear — APPROVED** [review]
Minor edits for your consideration, otherwise lgtm

File: src/main/java/seedu/address/model/person/SocialMedia.java (hunk 1 above)
Comment (kenpaxtonlim) [reply src/main/java/seedu/address/model/person/SocialMedia.java:78]: Not needed.

File: src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java (hunk 2 above)
Comment (kenpaxtonlim) [reply src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java:108]: Will change in the next update.

**Pull request merged**